Новые правила, или очередная волна тирании. Общие правила.

Belfogor opened the pull request:
Является авторским видением правил и результатом многомесячной и кропотливой работы (нет). Частично является переработкой понравившихся правил с бея, https://wiki.baystation12.net/Rules, за исключением раздела "правила персонажа", который так и не нашел достаточных голосов поддержки среди педалей. 

С оригинальной статьей можно знакомиться тут. 
https://wiki.taucetistation.org/%D0%A3%D1%87%D0%B0%D1%81%D1%82%D0%BD%D0%B8%D0%BA:Belfogor


> Представляет вахтер в отставке. Профинансировано министерством репрессий и пыток.

Commit:
Есть тело, есть дело.

RichardJones1 commented:
плюс за то, что все по пунктам(цифрам)

dred1792 commented:
Я бы возможно добавил пункт к антагонистам о том, что спектр их знаний несколько шире, чем у обычного члена экипажа.

volas reviewed (changes requested):
Лучше разделить на два отдельных PR-а. По нумерации - она правда лишняя.  
При каждой правке будут новые номера, так что отсылаться по номеру бесполезно. Если и делать, то лучше силами разметки markdown https://guides.github.com/features/mastering-markdown/#examples  
Если и не нумерованной, то тоже лучше разметкой.

Commit:
Апдейт для тирана Уолеса. 

Тиранит деда как может. Заставляет заниматься всякой бесполезной хренотенью и делить свой ПР надвое.

Belfogor commented:
Я не понял чего ты там от меня хочешь с разметкой, да и не сильно хочу понимать. Все предельно ясно. Что касательно нумерации, это была еще идея Сквазарба, и как по мне она была неплохая. Собственно, идея в том, чтобы в ризонах можно было ссылаться на конкретные правила.

volas commented:
>Собственно, идея в том, чтобы в ризонах можно было ссылаться на конкретные правила.

А потом ты добавляешь между правилами еще одно, и вся нумерация едет. И вот в ризоне уже некорректный номер.

В общем ладно, норм, я потом еще подкорректирую.
